sentry-opentelemetry/sentry-opentelemetry-core/src/main/java/io/sentry/opentelemetry/SentrySpanProcessor.java

```diff
@@ -151,7 +151,7 @@ public void onEnd(final @NotNull ReadableSpan otelSpan) {
           .getLogger()
           .log(
               SentryLevel.DEBUG,
-              "Unable to find Sentry span for OpenTelemetry span %s (trace %s).",
+              "Unable to find Sentry span for OpenTelemetry span %s (trace %s). This may simply mean it is a Sentry request.",
```

Comment:
```suggestion
              "Unable to find Sentry span for OpenTelemetry span %s (trace %s). This may simply means it is a Sentry request.",
```

Reply:
Hmm, did you also mean to remove `may` from the sentence for this to be correct again? But then again it's not certain this is a Sentry request, so `may` should stay. Pretty sure `means` is correct then.